execution_engine_testing/tests/benches/transfer_bench.rs

```diff
@@ -309,10 +341,22 @@ fn transfer_to_account_multiple_native_transfers(
             exec_request.protocol_version,
             exec_request.proposer.clone(),
         );
-        let builder = builder.exec(request).expect_success(); // flush to disk only after entire set
-        builder.commit();
+        if use_scratch {
+            builder.scratch_exec(request).expect_success();
+        } else {
+            builder.exec(request).expect_success();
+            builder.commit();
+        }
+    }
+    if use_scratch {
+        builder.write_scratch_to_lmdb();
```

Comment:
Why not include that in the previous `if use_scratch { ... }`?

Reply:
The previous `if` was in a loop - this needs to be run after the "block" has been completed and the cache is full of values ready to be written back.